src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
+import {
+  // BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import RouteAdmin from './Components/Admin/RouteAdmin';
+import UsersPage from './user';
 
 function App() {
```

Comment:
- [ ] I am still not able to see hotels, make reservations, or log out if I run the app according to your readme's instructions.  This is because the database's tables are empty.  Kindly provide a way to populate your users' database (like a seeds.rb file), so hotels can be shown, and reservations can take place.  This is the standard in the industry.
  
This was mentioned also by the previous reviewer.

## Explanation

- Several people will come to your repo to test your app, including other reviewers, the final evaluator, and potential employers.  We want them to be able to test your app easily and smoothly so you can showcase your new abilities.  

- We are here to help.  Our mission is to ensure your project's success in the final evaluation and with potential employers that might come to test your app. 

- Of course, since this is a special kind of review, you don't need our approval and may choose to ignore any of our suggestions, however, I highly suggest you make your app function as intended (e.g. showing some hotels to reserve).  That way, we ensure you demonstrate your skills properly so you can achieve success in the next and future steps.

## Tips and extra help

Look at these 2 articles that guide you to seed your database properly:

1- https://blog.devgenius.io/how-to-seed-with-ruby-on-rails-%EF%B8%8F-1d2dceda3e7d)
2- https://ninjadevel.com/seeding-database-ruby-on-rails/

As you can see, it's pretty simple here is an example with movies:

```ruby

# db/seeds.rb

Movie.destroy_all

Movie.create!([{
  title: "Soul",
  director: "Pete Docter",
  storyline: "After landing the gig of a lifetime, a New York jazz pianist suddenly finds himself trapped in a strange land between Earth and the afterlife.",
  watched_on: 1.week.ago
},
{
  title: "The Lord of the Rings: The Fellowship of the Ring",
  director: "Peter Jackson",
  storyline: "The Fellowship of the Ring embark on a journey to destroy the One Ring and end Sauron's reign over Middle-earth. A young Hobbit known as Frodo has been thrown on an amazing adventure, when he is appointed the job of destroying the One Ring, which was created by the Dark Lord Sauron.",
  watched_on: 2.years.ago
},
{
  title: "Terminator 2",
  director: "James Cameron",
  storyline: "Terminator 2 follows Sarah Connor and her ten-year-old son John as they are pursued by a new, more advanced Terminator: the liquid metal, shapeshifting T-1000, sent back in time to kill John Connor and prevent him from becoming the leader of the human resistance.",
  watched_on: 3.years.ago
}])

p "Created #{Movie.count} movies"
```

Looking at the example:

1- We clean the table with `Movie.destroy_all` so every time `rails db:seed` is run, the table is in a clean state.
2- We use the Active Record method `Movie.create!`.  Its only argument is an array of hashes with all your movies and their properties (in your case, those would be cars).
3- With the expression `p "Created #{Movie.count} movies"`, we inform the user that the command was successful, and the number of items created.
4- Lastly, we instruct your users on your backend readme, to run the required command `rails db:seed` after the migrations.

In the software industry, we _always_ seed our database and include proper instructions in all readme files, to prevent the delivery of empty apps to stakeholders.

We really hope this is helpful, and makes sense to you. 💪

Reply:
Hi @Meltrust, Thank you so much for your detail explanation. It is really helpful.